src/components/SideNav.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+// Import MUI Components
+import Box from "@mui/material/Box";
+import Drawer from "@mui/material/Drawer";
+import Divider from "@mui/material/Divider";
+import IconButton from "@mui/material/IconButton";
+import List from "@mui/material/List";
+import MenuIcon from "@mui/icons-material/Menu";
+import Typography from "@mui/material/Typography";
+
+export default function SideNav({ children }: any) {
```

Comment:
This shouldn't be of type 'any'. ReactNode is probably a better option here. More info here: https://www.carlrippon.com/react-children-with-typescript/